docs/azure/sdk/snippets/unit-testing/ResourceManager/ArmClient.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.ComponentModel;
+using System.Collections.Concurrent;
+
+#nullable disable
```

Comment:
Why are you disabling nullable? If this could be avoided, that would be preferrable.

Reply:
well I just added this to disable the warnings across all this file and others because there are properties that are not initialized in every ctors (which is expected). For example:
```csharp
public class Foo
{
    public string Name;
    protected Foo() {} // this is a ctor that makes sure we could mock this class, and compiler gives you warnings on property Name because it is not initialized in this ctor
}
```
disable nullable will remove the warning.
This is the same as we are having in all of our generated SDK code, for example:
https://github.com/Azure/azure-sdk-for-net/blob/802f045b3d3c6c1d1355277208790eb31e0efa1f/sdk/compute/Azure.ResourceManager.Compute/src/Generated/ArmComputeModelFactory.cs#L6
This line exists in every generated file of our SDK.

Since this file is not part of the document, and the reader of the document never sees it, I assume this might be OK?